api/services/contact_fields_mapping_service.py

```diff
@@ -16,19 +16,20 @@ def handle_contact_fields_data(self, jsonData):
             if not user_details:
                 return False
 
-            custom_fields = jsonData["contact"]["fields"]
-
-            for field_name, field_value in custom_fields.items():
-                if field_value:
-                    user_contact_field_mapping = (
-                        models.ContactFieldsMapping.query.get_by_field_name(
-                            field_name, field_value
-                        )
-                    )
-                    if user_contact_field_mapping:
-                        self.process_contact_fields_data(
-                            user_contact_field_mapping, user_details
+            field_names = db.session.query(models.ContactFieldsMapping.field_name).all()
```

Comment:
Instead of just taking the field names can't we fetch all custom field mappings? That will reduce the extra query that we are making on line 25

Reply:
okay